profile.md

```diff
@@ -0,0 +1,173 @@
+# Open Service Broker API Profile
+
+## Abstract
+
+The [Open Service Broker API specification](spec.md) allows for extensions
+and variations based on the environments in which it is being used; this
+document contains the recommended usage pattern for some of those variants.
+
+## Table of Contents
+
+- [Notations and Terminology](#notations-and-terminology)
+- [Context Object](#context-object)
+  - [Context Object Properties](#context-object-properties)
+  - [Cloud Foundry](#cloud-foundry)
+  - [Kubernetes](#kubernetes)
+
+## Notations and Terminology
+
+### Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to
+be interpreted as described in [RFC 2119]( https://tools.ietf.org/html/rfc2119).
+
+### Terminology
+
+The terminology defined below are defined by the
+[Open Service Broker API specification](spec.md) and are included here
+for convinience. If there are any inconsistencies between their definitions in
+this document and the [Open Service Broker API specification](spec.md), then
+the [Open Service Broker API specification](spec.md) SHALL take precidence.
+
+- *Platform*: The software that will manage the cloud environment into which
+  Applications and Service Brokers are provisioned.  Users will not directly
+  provision Services from Service Brokers, rather they will ask the Platform
+  (ie. their cloud provider) to manage Services and interact with the
+  Service Brokers for them.
+
+- *Service*: A managed software offering that can be used by an Application.
+  Typically, Services will expose some API that can be invoked to perform
+  some action. However, there can also be non-interactive Services that can
+  perform the desired actions without direct prompting from the Application.
+
+- *Service Broker*: Service Brokers manage the lifecycle of Services. Platforms
+  interact with Service Brokers to provision, bind and delete Service
+  Instances.
```

Comment:
I'd like to keep it like that in this terminology section to help identity them as special words, but whether we do that for the rest of the doc (or the spec) is less important to me.